chromium_src/components/signin/public/identity_manager/identity_manager_builder.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_SIGNIN_PUBLIC_IDENTITY_MANAGER_IDENTITY_MANAGER_BUILDER_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_SIGNIN_PUBLIC_IDENTITY_MANAGER_IDENTITY_MANAGER_BUILDER_H_
+
+#include <memory>
+
+#include "components/signin/public/identity_manager/identity_manager.h"
+
+namespace signin {
+
+enum class AccountConsistencyMethod;
```

Comment:
Needed?

Reply:
No, this  line is forgotten to delete, fixed at https://github.com/brave/brave-core/pull/7346/commits/332fa36d2755148c7a770737cf63551d709ba9ed . Thanks for finding this.